modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/image_selector/js/ImageSelectorImage.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React, {useEffect, useRef, useState} from 'react';
+
+const STR_BOTH = 'both';
+const STR_HORIZONTAL = 'horizontal';
+const STR_VERTICAL = 'vertical';
+
+const noop = () => {};
+
+const ImageSelectorImage = ({
+	direction = STR_VERTICAL,
+	imageSrc,
+	isCroppable,
+	onImageCrop = noop,
+	portletNamespace,
+}) => {
+	const containerRef = useRef();
+	const imageRef = useRef();
+
+	const [imagePosition, setImagePosition] = useState({x: 0, y: 0});
+	const [imageRelativePosition, setImageRelativePosition] = useState();
+	const [isDragging, setIsDragging] = useState();
+
+	const getImageStyle = () => {
+		let imageStyle = {};
+
+		if (isCroppable) {
+			imageStyle = {
+				cursor: isDragging ? 'move' : 'unset',
+				left: imagePosition.x,
+				position: 'relative',
+				top: imagePosition.y,
+			};
+		}
+
+		return imageStyle;
+	};
+
+	const handleMouseDown = (event) => {
+		event.stopPropagation();
+		event.preventDefault();
+
+		const imageContainer = containerRef.current;
+		const position = imageContainer.getBoundingClientRect();
+
+		setImageRelativePosition({
+			x: event.pageX - position.left,
+			y: event.pageY - position.top,
+		});
+
+		setIsDragging(true);
+	};
+
+	const handleMouseMove = (event) => {
+		event.stopPropagation();
+		event.preventDefault();
+
+		const imageContainer = containerRef.current;
+		const image = imageRef.current;
+
+		const imageContainerPosition = imageContainer.getBoundingClientRect();
+
+		let horizontalPos = imagePosition.x;
+		let verticalPos = imagePosition.y;
+
+		if (direction === STR_HORIZONTAL || direction === STR_BOTH) {
+			const horizontalDiff =
+				event.pageX -
+				imageContainerPosition.left -
+				imageRelativePosition.x;
+			horizontalPos = horizontalPos + horizontalDiff;
+
+			if (
+				horizontalPos >= 0 ||
+				horizontalPos < imageContainer.offsetWidth - image.offsetWidth
+			) {
+				event.preventDefault();
```

Comment:
I think that this event.preventDefault is redundant because we prevented it at the beginning of the function, but it is not dangerous we remove it in the next task.

Reply:
Yup, we can get rid of it on further prs replacing the old AUI component with this new ImageSelector one. Nice catch @boton !